### ISSUE TITLE
Use premake5 with MSCV on command line

### ISSUE THREAD

**Issue by PassWorld44** [premake/premake-core#1472]
I would like to use premake5 with MSCV (on the vs' command line) to compile, but I don't know how to do it.

I'm using premake5.

Can you help me?

**Comment by starkos** [comment]
It's unclear if you're trying to build Premake itself or a project which you've generated using Premake. Either way, you can build a Visual Studio solution from the command line like this:

```sh
devenv /build Release Solution.sln
```

Instructions for building Premake are [here](https://github.com/premake/premake-core/wiki/Building-Premake). Instructions for using Premake to generate Visual Studio solutions are [here](https://github.com/premake/premake-core/wiki/Using-Premake).

Hope that helps. I'm going to go ahead and close this issue but feel free to continue the conversation if you still have questions.

**Comment by PassWorld44** [comment]
Thanks for your answer!
But I wasn't clear enough... 

I did use a release of premake, so it's ok for this. I succeed to generate vs solutions too.

I was trying to build a personal project, using premake, and trying to compile it with the vs' compilator, with the vs' command line, but I don't know how to do it. 

Is MSCV taking .sln or makefiles to compile?

In fact, I don't have a very powerful computer, and it's lagging with visual studio, so I'm trying to code on sublime text, and compile with MSCV with the command line, but I have no idea how to use it to build the project that I have defined using premake.

I hope this is clear enough this time
